worker/operations.py

```diff
@@ -216,6 +306,7 @@ def pdf_to_text(*args, **kwargs):
         description=document_json.get("description", None),
         filename=filename,
         text=text,
+        assets=assets,
```

Comment:
since `skema_extraction` doesn't return `assets` in the case it is invoked this will error out with an `assets` not defined